pyproject.toml

```diff
@@ -38,6 +38,13 @@ dask-distributed = [
     "dask[distributed]>=2024.7.1",
     "bokeh!=3.0.*,>=2.4.2",
 ]
+latlon-domain-crop = [
+    "spherical-geometry>=1.3.2",
+    "cartopy>=0.23.0",
+    "gcsfs>=2024.10.0",
+    "jinja2>=3.1.4",
+]
```

Comment:
Are these really optional as is? It seems I can't really import and use any part of mdp without these right now. I don't see any problem with making them part of the normal requirements.

Reply:
Yea I think it is true that many users would not need these tools, but making the dependencies truly optional (making sure no imports fail) might be more effort than it is worth. Up to you.